convert2rhel/logger.py

```diff
@@ -88,6 +97,41 @@ def initialize_logger(log_name, log_dir=LOG_DIR):
     logger.addHandler(handler)
 
 
+def archive_old_logger_files(log_name):
+    """Archive the old log files to not mess with multiple runs outputs.
+    Every time a new run begins, this method will be called to archive the previous logs
+    if there is a `convert2rhel.log` file there, it will be archived using
+    the same name for the log file, but having an appended timestamp to it.
+        log_name = the name for the log file
+
+    For example:
+        /var/log/convert2rhel/archive/convert2rhel-1635162445070567607.log
+        /var/log/convert2rhel/archive/convert2rhel-1635162478219820043.log
+
+    This way, the user can track the logs for each run individually based on the timestamp.
+    """
+
+    current_log_file = "%s/%s" % (LOG_DIR, log_name)
```

Comment:
Sorry I didn't catch this earlier.  I just noticed that when we call `setup_logger_handler`, we pass in `log_dir`.  `log_dir` is then used to set what directory the current_log_file is output to here: https://github.com/oamg/convert2rhel/pull/359/files#diff-f82efabf7bb126100afd0472e3d8a5902da6576c6ddf75e3184cce9eb647c52aL83 

I recommend passing log_dir into this function too and setting both current_log_file and archive_log_dir based on that. ie:

``` python
current_log_file = os.path.join(log_dir, log_name)
archive_log_dir = os.path.join(log_dir, 'archive')
```

You can then also remove the toplevel ARCHIVE_LOG_DIR constant since we would be setting archive_log_dir based on log_dir.

Reply:
Nice catch, @abadger! Didn't see that either. Just fixed